### PR TITLE
Added Nether Labels to menu-themes

### PR DESCRIPTION
**Pull request by elfi-ox**

### CODE REVIEW

**Review by yar2000T — CHANGES_REQUESTED** [review]
Add your theme description and preview image to README.md. Also add file REUSE.toml (read readme file, it have detailed description)

**Comment by elfi-ox** [comment]
it should be good now

File: themes/nether-labels/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!--
+SPDX-FileCopyrightText: Elfi Ox <https://github.com/elfi-ox>
+SPDX-License-Identifier: CC-BY-1.0
```
Comment (Schneegans) [inline themes/nether-labels/README.md:3]: I think [you should use 4.0](https://creativecommons.org/licenses/by/1.0/deed.de).

```suggestion
SPDX-License-Identifier: CC-BY-4.0
```

**Review by Schneegans — CHANGES_REQUESTED** [review]
Looks good! I think only the one license identifier is off.

**Review by Schneegans — APPROVED** [review]

**Comment by Schneegans** [comment]
Awesome! Thank you very much!

**Pull request merged**